recipes/cwltest/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: cwltest
+  version: '1.0.20160907111242'
+
+source:
+  fn: cwltest-1.0.20160907111242.tar.gz
+  url: https://pypi.python.org/packages/16/85/e5cea003fb14a52c837de47b4ccf46b822c66a1bc2fb2bc5ac663a82919e/cwltest-1.0.20160907111242.tar.gz
+  sha256: 312c6069341304f4ec3d003b7d27a2e8d0fa4af93abe31af1177e92f9c905acd
+
+
+build:
+  number: 0
+  skip: True # [not py27]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - schema-salad ==1.14.20160708181155
+    - typing
+
+  run:
+    - python
+    - setuptools
```

Comment:
Sometimes packages depend on certain setuptools modules to find resources. But I don't know if it is the case here. @roryk are you aware of that?